WalletWasabi.Fluent/ViewModels/Wallets/Buy/BuyViewModel.cs

```diff
@@ -108,6 +108,8 @@ protected override void OnNavigatedTo(bool inHistory, CompositeDisposable dispos
 	{
 		base.OnNavigatedTo(inHistory, disposables);
 
+		SelectedOrder = Orders.FirstOrDefault();
```

Comment:
Doesn't this open the door for NRE?

Reply:
It doesn't change anything, the field was already Nullable